conpot/tests/empty_template.xml

```diff
@@ -1,10 +0,0 @@
-<conpot_template name="empty" description="empty">
```

Comment:
Just a template to show how the templates are to be created :slightly_smiling_face:

Reply:
I'm in favor of removing it for now considering that we are going forward with #534 Eventually this should be in the documentation.